src/mainwindow.cpp

```diff
@@ -121,10 +122,40 @@ MainWindow::MainWindow(const QString &fileName, QWidget *parent) :
     m_taskbarButton(nullptr),
     m_monospaceFont(false)
 {
-    ui->setupUi(this);
-
     QSettings settings;
 
+    bool useDarkMode = settings.value("darkMode", 0).toBool();
+#ifdef Q_OS_WIN
+    QSettings s("HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize", QSettings::NativeFormat);
+    if (s.value("AppsUseLightTheme") == 0)
+        useDarkMode = true;
```

Comment:
You override the setting here. it means that if Windows theme is set to dark, the user can't change the setting to light in loglite.